app.py

```diff
@@ -144,7 +155,10 @@ def render_and_submit_path(self, path, fields, first_frame, last_frame, sg_publi
                                       first_frame, last_frame,
                                       fields.get("version", 0), 
                                       fields.get("name", "Unnamed"),
-                                      color_space)
+                                      color_space, fields)
+
+        sub_first_frame = fields["sg_head_in"] if fields["sg_head_in"] else first_frame
+        sub_last_frame = fields["sg_tail_out"] if fields["sg_tail_out"] else last_frame
```

Comment:
As discussed last night, head_in/tail_out should be queried in tk-multi-publish and passed to tk-multi-reviewsubmission as first_frame/last_frame.